### ISSUE TITLE
Bug: Default settings are not loaded in the project

### ISSUE THREAD

**Issue by koladev32** [koladev32/djangorestframework-simple-apikey#25]
Default settings are not loaded in the project. When there is no `SIMPLE_API_KEY` setting provided, Django will throw an error. 

```
  File "/Users/koladev/projects/test-project/venv/lib/python3.11/site-packages/rest_framework_simple_api_key/models.py", line 15, in APIKeyManager
    key_crypto = ApiKeyCrypto()
                 ^^^^^^^^^^^^^^
  File "/Users/koladev/projects/test-project/venv/lib/python3.11/site-packages/rest_framework_simple_api_key/crypto.py", line 20, in __init__
    fernet_key, api_key_lifetime = settings.SIMPLE_API_KEY.get(
                                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/koladev/projects/test-project/venv/lib/python3.11/site-packages/django/conf/__init__.py", line 94, in __getattr__
    val = getattr(_wrapped, name)
          ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Settings' object has no attribute 'SIMPLE_API_KEY'
```

**Comment by cgelb** [comment]
I'm a new contributor and I would like to work on this issue.
How can I recreate the error you posted above?

**Comment by koladev32** [comment]
safsfasdf

**Comment by koladev32** [comment]
Hi @cgelb 

the issue has already been addressed in this PR #44 

In the future, we will try to be more precise in the description of the issues and how the fixes are going. 
Please, feel free to check any other issues in the project. :)
